### PR TITLE
[ja] add doc for cgroup

### LINKED ISSUE #38922: [ja] Translate concepts/architecture/cgroups into Japanese
**This is a Feature Request**

**What would you like to be added**

Translate https://kubernetes.io/docs/concepts/architecture/cgroups/ into Japanese

**Why is this needed**

**Comments**

/language ja
/assign

### PR DESCRIPTION
**Pull request by FeLvi-zzz**
Add desrtiption of cgroup.
https://kubernetes.io/docs/concepts/architecture/cgroups/

close #38922

### CODE REVIEW

**Comment by atoato88** [comment]
The target page on preview site:
https://deploy-preview-39085--kubernetes-io-main-staging.netlify.app/ja/docs/concepts/architecture/cgroups/

**Comment by atoato88** [comment]
/lgtm

It looks perfect for me :smile:

File: content/ja/docs/concepts/architecture/cgroups.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+cgroup v2では、`cgroup2fs`と出力されます。
+
+cgroup v1では、`tmpfs`と出力されます。
+
+## {{% heading "whatsnext" %}}
+
+- [cgroups](https://man7.org/linux/man-pages/man7/cgroups.7.html)についてもっと学習しましょう。
+- [コンテナランタイム](/ja/docs/concepts/architecture/cri)についてもっと学習しましょう。
+- [cgroupドライバー](/docs/setup/production-environment/container-runtimes#cgroup-drivers)についてもっと学習しましょう。
```
Comment (atoato88) [inline content/ja/docs/concepts/architecture/cgroups.md:106]: I know this `cgroupドライバー` has `ja` page, but [the page](https://kubernetes.io/ja/docs/setup/production-environment/container-runtimes/#cgroup%E3%83%89%E3%83%A9%E3%82%A4%E3%83%90%E3%83%BC) looks pretty old one. Topic for cgroup v2 is not witten on that `ja` page.
That's why you link here to `en` page, right?

**Review by Arhell — COMMENTED** [review]
lgtm

File: content/ja/docs/concepts/architecture/cgroups.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+ノード上やコンテナ内からユーザーが直接cgroupファイルシステムにアクセスしない限り、cgroup v2に切り替えたときのユーザー体験に目立った違いはないはずです。
+
+cgroup v2はcgroup v1とは違うAPIを利用しているため、cgroupファイルシステムに直接アクセスしているアプリケーションはcgroup v2をサポートしている新しいバージョンに更新する必要があります。例えば、
+
+* サードパーティーの監視またはセキュリティエージェントはcgroupファイルシステムに依存していることがあります。
+ エージェントをcgroup v2をサポートしているバージョンに更新してください。
+* Podやコンテナを監視するために[cAdvisor](https://github.com/google/cadvisor)をスタンドアローンのDaemonSetとして起動している場合、v0.43.0以上に更新してください。
+* JDKを利用している場合、[cgroup v2を完全にサポートしている](https://bugs.openjdk.org/browse/JDK-8230305)JDK 11.0.16以降、またはJDK15以降を利用することが望ましいです。
+
+## Linux Nodeのcgroupバージョンを特定する  {#check-cgroup-version}
```
Comment (t-inu) [inline content/ja/docs/concepts/architecture/cgroups.md:89]:
```suggestion
## Linux Nodeのcgroupバージョンを特定する {#check-cgroup-version}
```
空白が複数あっても問題はないですが。

**Review by t-inu — COMMENTED** [review]
Please check my comments.

File: content/ja/docs/concepts/architecture/cgroups.md (hunk 1 above)
Comment (FeLvi-zzz) [reply content/ja/docs/concepts/architecture/cgroups.md:106]: Oh, I didn't know that...
I want to keep it `en`, and I'll open an issue about updating https://kubernetes.io/ja/docs/setup/production-environment/container-runtimes and this page!

File: content/ja/docs/concepts/architecture/cgroups.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+cgroup v2を使うおすすめの方法は、デフォルトでcgroup v2が有効で使うことができるLinuxディストリビューションを使うことです。
+
+あなたのディストリビューションがcgroup v2を使っているかどうかを確認するためには、[Linux Nodeのcgroupバージョンを特定する](#check-cgroup-version)を参照してください。
+
+### 必要要件 {#requirements}
+
+cgroup v2を使うには以下のような必要要件があります。
+
+* OSディストリビューションがcgroup v2が有効であること
```
[the remark this answers is not in the document]
Comment (FeLvi-zzz) [reply content/ja/docs/concepts/architecture/cgroups.md:47]: 「OSディストリビューションでcgroup v2が有効であること」で違和感ないので、こちら反映させていただきます！

File: content/ja/docs/concepts/architecture/cgroups.md (hunk 1 above)
Comment (FeLvi-zzz) [reply content/ja/docs/concepts/architecture/cgroups.md:106]: I opened this.
https://github.com/kubernetes/website/issues/39125

Comment (atoato88) [reply content/ja/docs/concepts/architecture/cgroups.md:106]: Thank you for action.
It will be so valuable work.

**Comment by FeLvi-zzz** [comment]
/assign @nasa9084
Please check them when you have the time.

**Review by nasa9084 — COMMENTED** [review]
/approve

**Pull request merged**